src/pages/FilesPage/FilesTabs/DatasourcesTable/Modals/EditModal.tsx

```diff
@@ -0,0 +1,49 @@
+import { Modal, Input, Form, Button } from 'antd';
+import React, { FC } from 'react';
+
+import { createDataSource } from '../../../../../api/dataSourceDataGateService';
+
+type EditModalProps = {
+  opened: boolean;
+  handleOk: () => void;
+  handleCancel: () => void;
+};
+
+export const EditModal: FC<EditModalProps> = function EditModalRenderer({
```

Comment:
Rename file to `EditDatasourceModal`
I strongly recommend to keep file, const and function names the same:
`export const EditDatasourceModal: FC<EditModalProps> = function EditDatasourceModal({`

function name - it is that what you will see in React Developer Console in Chrome
It will really helps later, when the application will grow and we will debug ui